make/autoconf/libraries.m4

```diff
@@ -125,6 +126,7 @@ AC_DEFUN_ONCE([LIB_SETUP_LIBRARIES],
   LIB_SETUP_HSDIS
   LIB_SETUP_LIBFFI
   LIB_SETUP_MISC_LIBS
+  LIB_SETUP_VMATH
```

Comment:
The function (and file) should be named after "sleef", not "vmath".

Reply:
Yes, it seems weird. But the library we want to built out is `libvmath.so` instead of `libsleef.so`. And we not only check the sleef library, but also the ARM SVE feature inside it. So using `VMATH` suffix is more reasonable to me. WDYT?

Reply:
As I said above, you should not mix the two together. Keep the library handling for libsleef. Move the march setting to where it belongs. And rename the files, functions and variables after this.

Reply:
OK, I see. It makes sense that the suffix name should be choosed mainly based on the real module name that is searched/checked in configure.

Reply:
This still needs fixing.

Reply:
Yes, I will fix this together with removing the SVE cflags which I need more time to handle it with a better way. Thanks!